<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>beach</class>
 <widget class="QDialog" name="beach">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>661</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>어디로 여행을 떠나시겠습니까?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>671</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>50</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>선택</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>지역명</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>도시명</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>해변명</string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>701</width>
     <height>501</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>지역명</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>도시명</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>해변명</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>660</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>돌아가기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>630</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>630</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>해변을 선택하였습니다.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
